subsys/bluetooth/controller/ll_sw/ull_slave.c

```diff
@@ -79,7 +79,13 @@ void ull_slave_setup(memq_link_t *link, struct node_rx_hdr *rx,
 	       sizeof(lll->data_chan_map));
 	lll->data_chan_count = util_ones_count_get(&lll->data_chan_map[0],
 			       sizeof(lll->data_chan_map));
+	if (lll->data_chan_count < 2) {
```

Comment:
Why is the compare '< 2' and not '< 1' ?

Reply:
@cvinayak The function "util ones_count_get" detects all bits, which will result in including 37 38 39 channels. When the channel map is equal to 0xE000000000, Zephyr will bypass the channel detection and use the broadcast channel for communication.

Should this be considered？

Reply:
This does not cause any fatal error in the implementation. Do send a pull request to add the checks.